private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -15,17 +15,28 @@ const mapStateToProps = (state: RootState) => {
   };
 };
 
-interface DashboardStates {
-  error: ErrorMessageProps | null;
-}
-
 interface DashboardProps {
   isAuthenticated: boolean;
   user?: UserType;
   authButtonMethod: () => Promise<void>;
+  setPage: (currentPage: string) => void;
 }
 
-class Dashboard extends React.Component<DashboardProps, DashboardStates> {
+const mapDispatchToProps = (dispatch: any) => {
+  return {
+    setPage: (currentPage: string) => {
+      dispatch(setPage(currentPage));
+    }
+  }
+}
+
+class Dashboard extends React.Component<DashboardProps, {}> {
+
+  constructor(props: DashboardProps){
+    super(props);
+    props.setPage("Dashboard");
```

Comment:
Do you want every time a Dashboard component is created to change the page to dashboard?

Reply:
Oh I see, this is just the title.  Not the rendering children.

Reply:
Yes, we are only rendering the title page name, I will add some comments to make this more clear